### PR TITLE
Update shrinkwrapped express to v4.21.1

### PR DESCRIPTION
**Pull request by oskarwilliams**
Change-type: patch

Update shrinkwrapped express to v4.21.1 to bump cookie version to 0.7.1 to resolve low vulnerability

Resolves: https://github.com/balena-io/balena-cli/issues/2888
Change-type: patch

---

Please check the CONTRIBUTING.md file for relevant information and some
guidance. Keep in mind that the CLI is a cross-platform application that runs
on Windows, macOS and Linux. Tests will be automatically run by balena CI on
all three operating systems, but this will only help if you have added test
code that exercises the modified or added feature code.

Note that each commit message (currently only the first line) will be
automatically copied to the CHANGELOG.md file, so try writing it in a way
that describes the feature or fix for CLI users.

If there isn't a linked issue or if the linked issue doesn't quite match the
PR, please add a PR description to explain its purpose or the features that it
implements. Adding PR comments to blocks of code that aren't self explanatory
usually helps with the review process.

If the PR introduces security considerations or affects the development, build
or release process, please be sure to highlight this in the PR description.

Thank you very much for your contribution!

### CODE REVIEW

**Comment by oskarwilliams** [comment]
@alexgg Could I possibly get a quick eye on this? :)

**Comment by alexgg** [comment]
hey @oskarwilliams pinging some of the CLI maintainers for review @thgreasi @myarmolinsky

**Comment by myarmolinsky** [comment]
Hi, thank you for the PR, and sorry for missing it!

The commit makes sense to me, could you please squash both commits and push again? Please also run `npm dedupe` and add a commit for that change if there is any (as CI will fail if there is a change from the dedupe that has not been added to the PR)

**Comment by oskarwilliams** [comment]
Thank you very much @alexgg, @myarmolinsky I have hopefully done as asked! :)

**Comment by oskarwilliams** [comment]
I think some automatic test failed due to GitHub runner automated issues, can we retrigger the failed jobs?

**Comment by myarmolinsky** [comment]
> I think some automatic test failed due to GitHub runner automated issues, can we retrigger the failed jobs?

The failed `Test custom` is a flake, haven't seen the cancelled tests issue before, checking with our CI maintainers. We may need to update our macos version in the `flowzone.yml` (if we do so, it will be in a separate PR). I'll share and re-trigger tests when we know more

**Comment by myarmolinsky** [comment]
Once this PR merges: https://github.com/balena-io/balena-cli/pull/2893
We will request that you please rebase again (the dedupe commit could presumably be dropped since its changes will be included in that PR, unless another change to a package is made in the interim) and we will re-run the workflows

**Comment by myarmolinsky** [comment]
Please go ahead and rebase again when you have a chance. As noted, I expect you should be able to drop the `npm dedupe` commit. I will re-run workflows once the PR is rebased

**Comment by oskarwilliams** [comment]
Thanks @myarmolinsky, I have rebased and rerun dedupe

**Review by myarmolinsky — APPROVED** [review]

**Comment by myarmolinsky** [comment]
Thank you for your contribution! Apologies for the rocky road along the way, but we appreciate your perseverance!

**Pull request merged**
